_changelog/entries/2021/2021-03-15-zoom-v1-fixed-schema-report-meetings-webinars.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Zoom (v1) update: Fixed schema for `report_meetings` and `webinars`
```

Comment:
```suggestion
title: "Zoom (v1) update: Updated data for `report_meetings` and `webinars` tables"
```

Reply:
There were also some new fields tacked onto `report_webinars` fields, so this should be a bit more general than `fix`.

Also - make sure you check the pull requests to get the correct table names. I know the PR is titled `Fix schema for report meetings and webinars `, but here the `report` applies to both meetings AND webinars. You can see this is the case if you look at the **Files changed** tab of the PR.